example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -192,7 +221,8 @@ MM_CollectorLanguageInterfaceImpl::scavenger_masterSetupForGC(MM_EnvironmentBase
 void
 MM_CollectorLanguageInterfaceImpl::scavenger_workerSetupForGC_clearEnvironmentLangStats(MM_EnvironmentBase *env)
 {
-	/* Do nothing for now */
+	MM_EnvironmentStandard *envStd = (MM_EnvironmentStandard *)env;
+	MM_SublistFragment::flush((J9VMGC_SublistFragment*)&envStd->_scavengerRememberedSet);
```

Comment:
Fixed as you suggest: Implemented flush for remembered set fragment in MM_CollectorLanguageInterfaceImpl::flushNonAllocationCaches().